models/mongo/conversations.go

```diff
@@ -0,0 +1,63 @@
+// Package mongo is an implementation of models.AccountsRepository
+// that persists data on a MongoDB database.
+package mongo
+
+import (
+	"accounts-service/models"
+	"context"
+
+	// "errors"
+
+	// "github.com/google/uuid"
+	// "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	// "go.mongodb.org/mongo-driver/mongo/options"
+	"go.uber.org/zap"
+)
+
+type tchatsRepository struct {
```

Comment:
I think we should be consistent in the way we name things and call this repository `conversationsRepository`